src/Language/PureScript/Kinds.hs

```diff
@@ -20,6 +22,8 @@ data Kind
   | FunKind Kind Kind
   -- | Type-level strings
   | Symbol
+  -- | A kind constructor
+  | KindConstructor (Qualified (ProperName 'KindName))
```

Comment:
Could we just call this `NamedKind` or `OtherKind`? Constructor seems to imply that we can have kinds which take other kinds as arguments, which isn't true.